<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kompasSettings</class>
 <widget class="QDialog" name="kompasSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="bit_stop">
     <item>
      <property name="text">
       <string>One Stop</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Two Stop</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>One and Half Stop</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="parity_c">
     <item>
      <property name="text">
       <string>NoParity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EvenParity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OddParity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SpaceParity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MarkParity</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Скорость передачи</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Стоп бит</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Четность</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Интерфейс</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="Interface_c">
     <item>
      <property name="text">
       <string>RS-232</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RS-485</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MIL-STD-1553</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="Com_n"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Имя com-порта</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Биты данных</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="speed">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="data_bits">
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_Revert">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="buttonRevert">
     <property name="text">
      <string>Revert</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
